libs/game/scenes.ts

```diff
@@ -160,8 +169,27 @@ namespace scene {
     //% blockId=cameratop block="camera top"
     //% group="Camera"
     //% help=scene/camera-top
+    //% deprecated=true
     export function cameraTop() {
         const scene = game.currentScene();
         return scene.camera.drawOffsetY;
     }
+
+    /**
+     * Returns the specified camera property
+     * @param property The property to get
+     */
+    //% blockId=getcameraproperty block="camera %property"
+    //% group="Camera"
+    //% field.defl=0
```

Comment:
don't need this, it should default to the first value automatically

Reply:
Removed.